### PR TITLE
[luci-interpreter]Add negative cases for If kernel

### PR DESCRIPTION
**Pull request by struss**
For #1623 

This commit add negative cases for  `If` kernel on luci-interpreter.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

File: compiler/luci-interpreter/src/kernels/If.test.cpp [hunk 1]
```diff
@@ -88,9 +88,9 @@ TEST(IfTest, CondTrue)
   EXPECT_THAT(extractTensorData<float>(output), ElementsAreArray(ArrayFloatNear({6, 9})));
 }
 
-TEST(IfTest, CondFalse)
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/If.test.cpp:91]: It seems that this test (`CondFalse`) is removed. Is it intended?

Comment (struss) [reply compiler/luci-interpreter/src/kernels/If.test.cpp:91]: Oh, That's my mistake. I will fix it.

**Review by jinevening — APPROVED** [review]
LGTM

**Review by s-barannikov — APPROVED** [review]
LGTM

**Pull request merged**